packages/t/tobiaslocker_base64/xmake.lua

```diff
@@ -7,7 +7,7 @@ package("tobiaslocker_base64")
     add_urls("https://github.com/tobiaslocker/base64.git")
     add_versions("2024.02.26", "387b32f337b83d358ac1ffe574e596ba99c41d31")
 
-    on_install("windows|!arm64", function (package)
+    on_install("!windows|arm64", function (package)
```

Comment:
```suggestion
    on_install("windows|!arm64", "linux", function (package)
```

Reply:
I would add this change but it would limit the package to only windows and linux.

Reply:
That's fine, it was already limited to just windows before, you're just adding linux for now.